### PR TITLE
created HW3

### PR DESCRIPTION
**Pull request by nickytagoe**

### CODE REVIEW

**Review by showmic09 — CHANGES_REQUESTED** [review]
Hi @nickytagoe, 
Can you please add the result files with this push? I mean 'spill0.error, spill0.out.....' and the other files

**Comment by nickytagoe** [comment]
> Hi @nickytagoe, Can you please add the result files with this push? I mean 'spill0.error, spill0.out.....' and the other files

Hello Dr. @showmic09  please I have submitted the changes. Thank you

File: Nicky/HW3/R_OSPool.submit [hunk 1]
```diff
@@ -0,0 +1,27 @@
+SingularityImage = "/cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo-r:3.5.0"
+
+# Specify your executable and arguments
+executable = spill_calculation.r
+arguments = $(spill)
+
+# Tell HTCondor what to name log, standard error, and standard output("screen output") files
+log = log/spill$(Process).log
+output = output/spill$(Process).out
```
Comment (showmic09) [inline Nicky/HW3/R_OSPool.submit:9]: Hi @nickytagoe, please change the output and error location to *errout* as mentioned in the HW3. Just make changes here..you don't need to run the calculations again.

**Review by showmic09 — APPROVED** [review]
LGTM. Good work @nickytagoe

**Pull request merged**